src/pretix/control/templates/pretixcontrol/organizers/detail.html

```diff
@@ -7,7 +7,7 @@
         {% blocktrans with name=organizer.name %}Organizer: {{ name }}{% endblocktrans %}
         <a href="{% url "control:organizer.edit" organizer=organizer.slug %}"
            class="btn btn-default">
-            <span class="fa fa-edit"></span>
+            <span class="fa fa-sliders"></span>
```

Comment:
Its not that it *must* or that we don't like changes, its just that I value consistency very highly :wink: